src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -1,5 +1,9 @@
 package mate.academy
 
-class LetterRemover {
-    // implement ...
+object LetterRemover {
+    @JvmStatic
+    fun removeLetter(input: String): CharArray {
```

Comment:
FYI. You may write similar simple functions in single line
f.e. fun removeLetter(input: String) = input.lowercase().replace('r', '!').toCharArray()